resources/visualizations/Top-10-User-Agent-Strings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
UserAgent should be UserAgent.raw -> Do this for any visualizations that aggregate on strings with a .raw field